src/main/resources/pdiresources/transformations/program-drop-temp-table.ktr

```diff
@@ -0,0 +1,539 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we want to "drop program" table as soon as the migration is complete?  What if we need to recover something after the migration is complete?

@psmatsinhe

Reply:
Hi Nathan all  change  requests are done